osu.Game/Graphics/Cursor/OsuTooltipContainer.cs

```diff
@@ -27,13 +27,18 @@ public OsuTooltipContainer(CursorContainer cursor)
 
         public partial class OsuTooltip : Tooltip
         {
+            private const float max_width = 1024;
```

Comment:
I think this is a too long. 500 is more in line with what I'd expect going by browsers.